.changeset/stupid-foxes-type.md

```diff
@@ -0,0 +1,12 @@
+---
+'@backstage/plugin-catalog-react': patch
+---
+
+Use the history API directly in `useEntityListProvider`.
+
+This replaces `useSearchParams`/`useNavigate`, since I found that they caused additional rerenders.
```

Comment:
On the same topic as before, I'd say keep changelogs first person plural or just skip the person altogether 😅  "... since it was found ..."